spec/inputs/graphite_spec.rb

```diff
@@ -1 +1,54 @@
 require "logstash/devutils/rspec/spec_helper"
+require "logstash/inputs/graphite"
+require "logstash/timestamp"
+require "time"
+
+describe LogStash::Inputs::Graphite do
+  before do
+    srand(RSpec.configuration.seed)
+  end
+
+  let(:host) { "127.0.0.1" }
+  let(:port) { rand(5000) + 1025 }
+  let(:queue) { [] }
+
+  let(:client) { TCPSocket.new(host, port) }
+
+  subject { LogStash::Inputs::Graphite.new("host" => host, "port" => port) }
```

Comment:
I wonder why not using the kind of input helper @colinsurprenant added in devutils? https://github.com/elastic/logstash-devutils/blob/master/lib/logstash/devutils/rspec/logstash_helpers.rb#L64